frontend/src/main/components/Quarters/SingleQuarterDropdown.js

```diff
@@ -22,13 +10,15 @@ function SingleQuarterDropdown({
   label = "Quarter",
 }) {
   const localSearchQuarter = localStorage.getItem(controlId);
+  const lastIndex = quarters.length - 1;
+  const defaultQuarter = quarters[lastIndex].yyyyq;
+
   if (!localSearchQuarter) {
-    localStorage.setItem(controlId, quarters[0].yyyyq);
+    localStorage.setItem(controlId, defaultQuarter);
   }
 
   const [quarterState, setQuarterState] = useState(
-    // Stryker disable next-line all : not sure how to test/mock local storage
-    quarter.yyyq || localSearchQuarter || quarters[0].yyyyq,
```

Comment:
You need to fix thee parameters to match. You use 'yyyq' when you should use 'yyyyq' throughout

Reply:
it's inherited from the past, i don't think it needs to be changed